docs/workflows/copy.rst

```diff
@@ -14,7 +14,8 @@ Like all Pulp repositories, you can use the ${repo_href}/modify/ endpoint to:
 
 * add or remove individual content units from a repository by HREF
 * roll back the content present in a repository to that of a previous version using 'base_version'
-* clone a repository version from another repo using 'base_version'
+* clone a repository version from another repo using 'base_version'. This operation will copy all
+  the content from 'base_version' into the repository.
```

Comment:
Actually hold up, this isn't true.

Reply:
With the basic modify/ endpoint, ```base_version``` will effectively clone the repository version specified into the current repo.  It doesn't copy, it replaces.